### PR TITLE
Features/ig.multiscene

### PR DESCRIPTION
**Pull request by gekidoslair**
Multiscene loader - edit mode UX as well as a runtime API for loading.

### CODE REVIEW

**Review by jeanblouin — APPROVED** [review]
Nice tool! Initially it was not obvious to me that you needed to add the scene in the build setting for loading the scene in play mode in the editor, but it makes sense because SceneManager.LoadScene requires it.
LGTM

**Comment by gekidoslair** [comment]
I'll update the docs to mention that!

File: com.unity.film-tv.toolbox/Samples/MultiScene.meta [hunk 1]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```
Comment (clusty) [inline com.unity.film-tv.toolbox/Samples/MultiScene.meta]: You are missing .samples.json. https://github.cds.internal.unity3d.com/unity/com.unity.package-starter-kit/blob/master/Samples/Example/.sample.json

Comment (clusty) [inline com.unity.film-tv.toolbox/Samples/MultiScene.meta:1]: You are missing .samples.json. for every sample https://github.cds.internal.unity3d.com/unity/com.unity.package-starter-kit/blob/master/Samples/Example/.sample.json

**Review by clusty — COMMENTED** [review]
General comments:
* Please don't expose public things unless they are really needed. And if you do expose them, they need to be documented (every single public symbol needs a documentation)

I generally prefer not exposing every data public directly, but rather through get/set. This allows you debug things (there is only one external way to access this data and putting a breakpoint there allows you to catch all access)

If you wish to go forward with the changes, I can put comments on all instances.

File: com.unity.film-tv.toolbox/Samples/MultiScene.meta (hunk 1 above)
Comment (gekidoslair) [reply com.unity.film-tv.toolbox/Samples/MultiScene.meta:1]: k done!

**Comment by gekidoslair** [comment]
> Nice tool! Initially it was not obvious to me that you needed to add the scene in the build setting for loading the scene in play mode in the editor, but it makes sense because SceneManager.LoadScene requires it.
> LGTM

added a note in the docs about this linking to the page in the manual

**Pull request merged**